web_full_stack/js/README.md

```diff
@@ -2,13 +2,13 @@ JavaScript Code Extension
 ---
 The JS Code Extension is a React application created using [create-react-app](https://github.com/facebook/create-react-app). Tests are written using [enzyme](https://github.com/airbnb/enzyme) and [react-router](https://github.com/ReactTraining/react-router) manages routing.
 
+# Setup
 You'll need [yarn](https://yarnpkg.com/en/) installed and a recent version (>= 8) of [Node](https://nodejs.org/en/) to use this project. If you're using homebrew you can brew install both of those or if you want to manage multiple versions of node on your machine, we suggest using a tool like [nodenv](https://github.com/nodenv/nodenv).
```

Comment:
Let's double check that this is still true. Might need a newer version with all the updates